api/src/pcapi/core/fraud/api.py

```diff
@@ -474,8 +474,17 @@ def _handle_duplicate(fraud_items: list[models.FraudItem], fraud_check: models.B
     if not duplicate_had_underage_credit:
         return
 
-    users_api.suspend_account(user, constants.SuspensionReason.FRAUD_SUSPICION, actor=None)
-    users_api.suspend_account(duplicate_beneficiary, constants.SuspensionReason.FRAUD_SUSPICION, actor=None)
+    users_api.suspend_account(
+        user,
+        constants.SuspensionReason.FRAUD_SUSPICION,
+        actor=None,
```

Comment:
**Détail :** pourquoi laisses-tu `actor=None` dans l'un et le supprimes-tu dans l'autre ?

Reply:
En fait mypy a répondu :
`src/pcapi/core/fraud/api.py:486:15: error: Argument "actor" to "suspend_account" has incompatible type "str"; expected "User | None"  [arg-type]`
Donc à corriger, ce n'est plus un détail :)

Reply:
C'est pratique la relecture !
Dans l'autre cas je me suis trompé, je devais remplir comment, et pas actor 😅

Reply:
Faut vraiment que je change ma version de mypy, la 1.4.1 bug tout le temps, donc je vois pas les erreurs avant...